File: web3rpc/rpc-specs/code-samples/javascript/eth/filter/uninstallFilter.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const filterId = "0xb";

  provider.eth
    .uninstallFilter(filterId, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/eth/filter/uninstallFilter.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

filterId = "0xb"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.uninstall_filter(filterId)

print(eth_response)
